MetaMask/metamask-extension
# test:e2e: create Security Alert Blockaid tests for signature requests

## **Description**

This PR creates tests for the Security Alert Blockaid alerts with the provided signatures from https://wobbly-nutmeg-8a5.notion.site/MM-E2E-Testing-1e51b617f79240a49cd3271565c6e12d

We will be adding tests for the remaining req/res in the above link in a separate PR

## **Manual testing steps**

1. run `yarn build:test:flask`
2. run `yarn test:e2e:single test/e2e/tests/confirm/security-alert-blockaid.spec.js --browser=firefox`

## **Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change._

### **Before**

_[screenshot]_

### **After**

_[screenshot]_

## **Related issues**

fixes: https://github.com/MetaMask/MetaMask-planning/issues/991

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
